### PR TITLE
Fix pai examples

### PR DESCRIPTION
**Pull request by SparkSnail**

### CODE REVIEW

**Comment by ultmaster** [comment]
Has it passed IT?

**Comment by SparkSnail** [comment]
> Has it passed IT?

There is no code change, it is doc and example change. IT does not test these example files.

File: examples/trials/nas_cifar10/config_paiYarn_ppo.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  command: sh ./macro_cifar10_pai.sh
+  codeDir: ./
+  gpuNum: 1
+  cpuNum: 1
+  memoryMB: 8196
+  image: msranni/nni:latest
+  virtualCluster: nni
+paiYarnConfig:
+  userName: your_account
+  passwd: your_passwd
```
Comment (ultmaster) [inline examples/trials/nas_cifar10/config_paiYarn_ppo.yml]: passwd or passWord or password?

File: examples/trials/mnist-advisor/config_pai.yml [hunk 2]
```diff
@@ -27,10 +27,13 @@ trial:
...
+  nniManagerNFSMountPath: /home/user/mnt
+  containerNFSMountPath: /mnt/data/user
+  paiStoragePlugin: team_wise
 paiConfig:
   #The username to login pai
   userName: username
-  #The password to login pai
-  passWord: password
+  #The token to login pai
+  token: token
```
Comment (xuehui1991) [inline examples/trials/mnist-advisor/config_pai.yml]: do we have a description of "where could I get the token"?

Comment (ultmaster) [reply examples/trials/mnist-advisor/config_pai.yml]: It's like github access token. It's achievable from PAI user settings. I think users can figure that out.

File: examples/trials/nas_cifar10/config_paiYarn_ppo.yml (hunk 1 above)
Comment (SparkSnail) [reply examples/trials/nas_cifar10/config_paiYarn_ppo.yml]: fixed.

File: examples/trials/mnist-advisor/config_pai.yml (hunk 2 above)
Comment (SparkSnail) [reply examples/trials/mnist-advisor/config_pai.yml]: the token tutorial is in https://github.com/microsoft/nni/pull/2011

**Pull request merged**